src/contacts/ContactViewer.js

```diff
@@ -113,14 +113,18 @@ export class ContactViewer {
 					m(".header.pt-ml", [
 						m(".contact-actions.flex-space-between.flex-wrap.mt-xs", [
 							m(".left.flex-grow-shrink-150", [
-								m(".h2.selectable.text-break", this.contactAppellation),
-								m(".flex-wrap.selectable", insertBetween([
+								m(".h2.selectable.text-break", [
+									this.contactAppellation,
+									m.trust("&nbsp;") // alignment in case nothing is present here
```

Comment:
We try to track and check each `m.trust()` usage, I think `\u00A0` should work as well?

Reply:
It does! I'll use that instead.